### PR TITLE
Dynamic configuration files

### LINKED ISSUE #3: Use a config instead of hardcoding it in the code
Currently, there are a lot of "channel id", "group id", "reaction id", .. and so on that are hardcoded.

I think it'd be better to be able to "configure" it, so that the bot could be used on other discords. :D

### PR DESCRIPTION
**Pull request by ricknox**
This fixes #3

### CODE REVIEW

File: src/util/config.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ * file that was distributed with this source code.
+ */
+
+const _ = require('lodash');
+const logger = require('./logger');
+const config = require('../../config.json');
+
+// As a best practice all global variables should be referenced via global.
+// Syntax and their names should always begin with g
+global.gConfig = config;
```
Comment (Pierstoval) [inline src/util/config.js]: Why should it be prefixed with a "g" ?

Comment (ricknox) [reply src/util/config.js]: as in globalConfig.

Comment (Pierstoval) [reply src/util/config.js]: Yeah, but why not simply `global.config` instead?

Comment (ricknox) [reply src/util/config.js]: Fixed in 49c8b2f5e7c0ac805ea7cd03983e3fd4c89f44ca.

**Comment by ricknox** [comment]
We could even go futher by ignoring `config.json` and only ship a empty `config.json.dist` file with a small installation guide.

**Comment by Pierstoval** [comment]
> We could even go futher by ignoring `config.json` and only ship a empty `config.json.dist` file with a small installation guide.

Totally agree 👍

File: config.json.dist [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "channels": {
+    },
```
Comment (Pierstoval) [inline config.json.dist]: I suggest you add `null` values to get a list of all channels and roles before setting up the project

**Pull request merged**